docs/API/API-socket-options.md

```diff
@@ -1068,20 +1070,22 @@ undefined behavior:
 
   - **Bidirectional in Caller-Listener arrangement**: it is recommended to use
   a rule whereby you will be setting the `PBKEYLEN` exclusively either on the
-  Listener or on the Caller. The value set on the Listener will win, if set on
-  both parties.
+  Listener (the service defines the encryption rules strictly) or on the Caller
+  (the service allows all clients to freely decide about encryption, or the
+  server uses more elaborate rules for encryption).
 
-  - **Bidirectional in Rendezvous arrangement**: you have to know the passphrases
-  for both parties, as well as `PBKEYLEN`. Set `PBKEYLEN` to the same value on
-  both parties (or leave the default value on both parties, which will
-  result in 16)
+  - **Bidirectional in Rendezvous mode**: In Rendezvous mode it is not possible for a party to be "unknown", 
+  as it is with Listener mode, so both parties should set the same passphrase and the same key length, 
+  or both should leave the `SRTO_PBKEYLEN` option unchanged (which results in default 16).
 
   - **Unwanted behavior cases**: if both parties set `PBKEYLEN` and the value
-  on both sides is different, the effective `PBKEYLEN` will be the one that is
-  set on the Responder party, which may also override the `PBKEYLEN` 32 set by
-  the sender to value 16 if such value was used by the receiver. The Responder
-  party is the Listener in a Caller-Listener arrangement. In Rendezvous it's a
-  matter of luck which party becomes the Responder.
+  on both sides is different, this is considered a conflict. It is resolved by
+  the Initiator party, which takes its own value, if the `SRTO_SENDER`
+  option is set to true. Otherwise, it takes the value from the Responder party. The
+  assignment of Initiator-Responder roles matches the Caller-Listener layout. In the case of Rendezvous the assignment is a matter of luck, resulting from the cookie comparison.
```

Comment:
But it's true. You can formulate it differently, but I think it should suffice to say "In the case of Rendezvous this assignment is according to the result of the cookie comparison and therefore unpredictable." @stevomatthews what do you think?

Reply:
I don't like the word "luck" in the initial suggestion because it does not look scientific.
The cookie is even not a completely random thing, but rather an (MD5) function over time, host IP and port.

Reply:
That's perfectly right, but "random" isn't defined as "taken out of an unknown source", but "with no predictable result", and this here is the case. Hence I even support saying it directly that it's unpredictable.

Reply:
The time is taken with a one-minute accuracy, making it quite predictable.

Reply:
Ok, I can make it extremely concise, like: "In the case of Rendezvous this assignment depends on the result of the cookie comparison". That's clear enough.

Reply:
@ethouris  I think your last suggestion is fine.